devicetypes/smartthings/fibaro-smoke-sensor.src/i18n/messages.properties

```diff
@@ -0,0 +1,1463 @@
+#  Copyright 2020 SmartThings
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy
+#  of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+
+# Device Preferences
+'''Excess temperature signaling interval'''.en=Excess temperature signalling interval
```

Comment:
Is `en` specifically provided or do we set it the same as `en-us` or `en-gb`? Using two `ll` is British English and I'd think we would default to American English (one `l`).

Reply:
this one was left as-is